releases/release-1.20/release-notes/maps/pr-93577-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 93577
+releasenote:
+  text: Updated Cluster Autoscaler to 1.19.0.
```

Comment:
👋🏼 @celestehorgan, not to nitpick an already-done work, but I want to make sure I can incorporate as much styling as possible into the incoming `alpha.2` which I will generate

I noticed that most of the fixes use infinitive verbs (i.e. fix, add, ...) or highlights transition (i.e. now _behavior_) -- should this one be "Update Cluster Autoscaler to 1.19.0"? or is there something I'm missing?